src/riak_core_tcp_mon.erl

```diff
@@ -458,7 +458,7 @@ ssl_test_() ->
         spawn(fun () ->
             %% server
             {ok, S} = ssl:transport_accept(LS),
-            ok = ssl:ssl_accept(S),
+            ok = ssl:handshake(S),
```

Comment:
http://erlang.org/doc/man/ssl.html#handshake-1 - looks like it cannot just return OK, will always return a socket

Reply:
Thanks. Updated. I might have not run the ssl suite locally :(